### PR TITLE
Introduce Circuit Breaker

### PR DESCRIPTION
**Pull request by AndrewOlenik**
#7:
- feign-hystrix has been introduced
- switch from com.netflix.feign to io.github.openfeign

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Assuming that the Docker image is already built on the previous step
 ## Tech Stack
 - [Spring Boot](https://projects.spring.io/spring-boot/) : Application framework
 - [Feign](https://github.com/OpenFeign/feign) : HTTP Client library
+  - [Feign-Hystrix](https://github.com/OpenFeign/feign/tree/master/hystrix) : wraps Feign's http requests in Hystrix, which enables the Circuit Breaker Pattern.
```
Comment (trioletas) [inline README.md]: formatting is off

Comment (trioletas) [inline README.md]: > enables the Circuit Breaker Pattern

let's add a link to [wiki](https://en.wikipedia.org/wiki/Circuit_breaker_design_pattern)

Comment (AndrewOlenik) [reply README.md]: Do you think it is required? Hystrix has intensive documentation

Comment (AndrewOlenik) [reply README.md]: re-phrase please

File: api/src/main/java/lv/ctco/tpl/bff/configuration/FeignConfiguration.java [hunk 2]
```diff
@@ -11,16 +17,39 @@
 @Configuration
 public class FeignConfiguration {
 
+    private static final int EXECUTION_TIMEOUT_IN_MILLIS = 2000;
```
[the remark this answers is not in the document]
Comment (AndrewOlenik) [reply api/src/main/java/lv/ctco/tpl/bff/configuration/FeignConfiguration.java]: thx, will do

File: README.md (hunk 1 above)
Comment (trioletas) [reply README.md]: did not realize Feign-Hystrix was a sub element of the Feign Tech Stack entry
please disregard

Comment (trioletas) [reply README.md]: fair enough

**Pull request merged**